src/main/flight/mixer.c

```diff
@@ -448,6 +449,9 @@ void loadLaunchControlMixer(void)
 
 #ifndef USE_QUAD_MIXER_ONLY
 
+static FAST_RAM_ZERO_INIT uint8_t revFlags[16];
```

Comment:
Some parts of your code are conditional to `USE_QUAD_MIXER_ONLY`, other parts aren't. This needs cleaning up. We might want to consider a conditional of its own for this code, to allow users with MCUs that are short on flash space to build targets without it.

Reply:
simple uint16_t bitmask will do just fine ... 

BTW: as implemented, uint8_t will overflow for motor >= 8

Reply:
@ledvinap revFlags[] just contains 0 or 1

Reply:
Corrections made!